src/renderer/NotificationRenderer.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import {NotificationStacked} from "./components/Notification/NotificationStacked";
+import {Notification} from "./components/Notification/Notification";
+import {connect} from "react-redux";
+import {bindActionCreators, Dispatch} from "redux";
+import {IRootState} from "./reducers/index";
+import {INotificationStateObject} from "./reducers/notification";
+import {removeNotificationAction} from "./actions/notification";
+
+const NotificationRendererContainer: React.FunctionComponent<
+IInjectedProps & Pick<IRootState, "notificationArray">> = (props) => {
+
+    const mapNotifications = (arrays: INotificationStateObject): React.ReactElement => {
+        return(
+            <>
+                <div className={"notification-stacked-container right bottom"}>
+                    {arrays.stacked.map((n, index) =>
+                        <NotificationStacked
+                            key={index}
+                            title={n.title}
+                            isVisible={n.isVisible}
+                            level={n.level}
+                            horizontalPosition={n.horizontalPosition}
+                            verticalPosition={n.verticalPosition}
+                            onClose={(): void => {props.removeNotification(index, true);}}
```

Comment:
It's probably safer, if each notification has some id(maybe id should include some container identification) instead of dynamic index